src/Microsoft.Identity.Client/OAuth2/OAuthConstants.cs

```diff
@@ -83,6 +83,59 @@ internal static class OAuth2Error
         public const string AuthorizationPending = "authorization_pending";
     }
 
+    internal static class OAuth2SubError
```

Comment:
>OAuth2SubError [](start = 26, length = 14)

Can we call these InvalidGrantClassifications ?

Reply:
This is an internal class and the name is taken from the server side (this is how they call them)